core-header-panel.html

```diff
@@ -139,12 +139,24 @@
        * @event scroll
        */
 
-      computed: {
-        dropShadowClass: 'calcDropShadowClass(atTop, mode, shadow)',
-        mainContainerClass: 'calcMainContainerClass(mode)'
-      },
+      properties: {
+
+        animateDuration: {
+          value: 200
+        },
+
+        atTop: {
+          value: true
```

Comment:
Default value of `true` is not supported for booleans, strictly speaking. Is this an effectively private property?